src/setComplement.js

```diff
@@ -0,0 +1,13 @@
+export default function setComplement(arr1, arr2) {
+
+  var resultsArray = []
+
+
+  arr2.filter(function(x){
+    if(arr1.indexOf(x) < 0){
```

Comment:
this can be changed to use the `includes` function. See here - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes